src/main/webapp/index.html

```diff
@@ -28,69 +28,23 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <script src="script.js"></script>
+    <script src="https://code.jquery.com/jquery-1.10.2.js"></script>
+    <script type="text/javascript">
+      !function(e,t,r,n){if(!e[n]){for(var a=e[n]=[],i=["survey","reset","config","init","set","get","event","identify","track","page","screen","group","alias"],s=0;s<i.length;s++){var c=i[s];a[c]=a[c]||function(e){return function(){var t=Array.prototype.slice.call(arguments);a.push([e,t])}}(c)}a.SNIPPET_VERSION="1.0.1";var o=t.createElement("script");o.type="text/javascript",o.async=!0,o.src="https://d2yyd1h5u9mauk.cloudfront.net/integrations/web/v1/library/"+r+"/"+n+".js";var l=t.getElementsByTagName("script")[0];l.parentNode.insertBefore(o,l)}}(window,document,"qdSQs62OlE99gePD","delighted");
```

Comment:
What is this function for? It is not very readable. If it is copied from somewhere, you could probably attribute to that code and import it rather than inlining it.

Reply:
It's something that Angela wanted me to include, it's a code snippet that makes a survey pop up at the bottom of the screen asking for a rating of satisfaction. It was generated by https://app.delighted.com/

Reply:
I think it is okay for now. Please attribute to the website either in readme or here with a comment.